src/components/ImageCard/ImageCard.stories.tsx

```diff
@@ -2,43 +2,94 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { select, text } from '@storybook/addon-knobs';
 
+import Listing from 'Listing';
+
 import ImageCard from './ImageCard';
 
-const ImageCardExample = () => {
+const SingleImageCard = () => {
   const authors = text(
     'authors',
     'Christiane Hertz-Fowler, Georgia Walton, Chonnettia Jones'
   );
 
   const date = text('Date', '03/08/1991');
 
+  const href = text('href', '/news/all');
+
   const imageAlt = text('Image alt text', 'Alternative image text');
   const imageSrc = text('Image path', `https://via.placeholder.com/300`);
 
+  const metaLabel = text('Meta Label', 'Climate Change');
+
   const title = text('Title', 'My section');
   const titleAs = select('Title element', ['h2', 'h3', 'h4', 'h5', 'h6'], 'h3');
 
-  const metaLabel = text('metaLabel', 'Meta');
-
-  const url = text('URL', '/news/all');
-
   return (
     <ImageCard
       authors={authors
         .trim()
         .split(',')
         .map(a => a.trim())}
       date={date}
+      href={href}
       imageAlt={imageAlt}
       imageSrc={imageSrc}
       metaLabel={metaLabel}
       title={title}
       titleAs={titleAs}
-      url={url}
     />
   );
 };
 
-const stories = storiesOf('Components|ImageCard', module);
+const MultipleImageCard = () => {
+  const cardCount = select('How many ImageCards?', [2, 3], 3, 'General');
+
+  const knobs = [...Array(cardCount).keys()].map(i => ({
+    authors: text(
+      'authors',
+      'Christiane Hertz-Fowler, Georgia Walton, Chonnettia Jones',
+      `Card ${i + 1}`
+    ),
+    date: text('Date', '03/08/1991', `Card ${i + 1}`),
+    href: text('href', '/news/all', `Card ${i + 1}`),
+    imageAlt: text('Image alt text', 'Alternative image text', `Card ${i + 1}`),
+    imageSrc: text(
+      'Image path',
+      `https://via.placeholder.com/300`,
+      `Card ${i + 1}`
+    ),
+    metaLabel: text('Meta Label', 'Climate Change', `Card ${i + 1}`),
+    title: text('Title', 'My section', `Card ${i + 1}`),
+    titleAs: select(
+      'Title element',
+      ['h2', 'h3', 'h4', 'h5', 'h6'],
```

Comment:
Minor: Playing around with this I noticed there's a difference in letter-spacing between the headings so the typography should prob have overrides

Reply:
Fixed in most recent commit (removed letter-spacing in line with Zeplin https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/screen/5f96dd2836478aa65797ec96)